src/hiero_sdk_python/tokens/token_pause_status.py

```
@@ -6,18 +6,21 @@
 from enum import Enum
 from typing import Any
 from hiero_sdk_python.hapi.services import basic_types_pb2
 
+
 class TokenPauseStatus(Enum):
     """
     Enumeration of token pause statuses:
 
-      • PAUSE_NOT_APPLICABLE – pause not relevant  
-      • PAUSED              – token is paused  
+      • PAUSE_NOT_APPLICABLE – pause not relevant
```

Ruff (0.14.14)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)